github/repos_contents.go

```diff
@@ -130,7 +130,8 @@ func (s *RepositoriesService) DownloadContents(owner, repo, filepath string, opt
 func (s *RepositoriesService) GetContents(owner, repo, path string, opt *RepositoryContentGetOptions) (fileContent *RepositoryContent, directoryContent []*RepositoryContent, resp *Response, err error) {
 	// escape characters not allowed in URL path.  This actually escapes a
 	// lot more, but seems to be harmless.
```

Comment:
With the change below, is this comment still accurate/relevant or should it be updated?